src/main/studio/qeditor/RSTokenMaker.java

```diff
@@ -14,6 +14,7 @@ public class RSTokenMaker extends TokenMakerBase {
     public static final String CONTENT_TYPE = "text/q";
 
     private QSyntaxParser parser = new QSyntaxParser();
+    public static boolean unindentCurlyBraces = false;
```

Comment:
I didn't find a code which would initialize the variable from the settings on start-up. It looks this variable is updated only if I call Settings Dialog.  

Also generally it is not encourage in the Java world to have public access to variables. Usually they are doing via setter functions...

Reply:
Updated, can you check again?

Reply:
I think that's good now.